src/sentry/event_manager.py

```diff
@@ -399,7 +399,8 @@ def save(self, project_id, raw=False, assume_normalized=False, start_time=None,
             "last_seen": job["event"].datetime,
             "first_seen": job["event"].datetime,
             "active_at": job["event"].datetime,
-            "data": group_metadata,
+            "metadata": dict(job["data"]["metadata"]),
+            "received_timestamp": job["received_timestamp"],
```

Comment:
Would it make sense to pass `metadata` and `received_timestamp` into `_save_aggregate` as regular arguments? It took me a while to see that they are not used directly for group creation like the other items in this dict.

Reply:
yeah makes sense